controllers/nodehealthcheck_controller_test.go

```diff
@@ -133,7 +133,11 @@ var _ = Describe("Node Health Check CR", func() {
 			})
 
 			It("fails creation on negative number", func() {
-				Skip("TODO find out how to validate a minimum for IntOrString")
+				Skip("Does not work yet")
```

Comment:
Just making sure it's not an accidental left over, are you planning a separate PR for that ?

Reply:
will be a seperate PR as soon as this validation is supported by controller-utils. Will add a comment to explain this better 👍🏼 (it's on the Jira ticket only at the moment)

Reply:
done